.github/actions/setup/action.yaml

```diff
@@ -0,0 +1,25 @@
+name: Setup Rust Builder
+description: 'Prepare Rust Build Environment'
+inputs:
+  cache-key:
+    description: 'the rust cache key suffix'
+    required: false
+    default: ''
+
+runs:
+  using: "composite"
+  steps:
+    - name: Cache Cargo
+      uses: actions/cache@v3
+      with:
+        path: |
+          ~/.cargo/bin/
+          ~/.cargo/registry/index/
+          ~/.cargo/registry/cache/
+          ~/.cargo/git/db/
+          ./frb_example/with_flutter/rust/target
+        key: ${{ runner.os }}-cargo-integrate-ios-${{ hashFiles('**/Cargo.lock') }}-${{ inputs.cache-key }}
+        restore-keys: |
+            ${{ runner.os }}-cargo-integrate-ios-${{ hashFiles('**/Cargo.lock') }}
+            ${{ runner.os }}-cargo-integrate-ios
```

Comment:
uber-nit: maybe no longer "integrate-ios", but should be something that is passed in as an argument?

Reply:
Not sure. If cache for different platforms will collide, then we should add this

Reply:
1. We do have arg -> ${{ runner.os }}
2. no fail happen for now. 
---
But you have a point
Curious. Could it have been needed before?
For now, I'm thinking about the fact that between different IOS, the previous product of rust, is the same with name,and would be overwritten
android in particular

Reply:
* android output
[2023-04-12T16:48:44Z INFO  cargo_ndk::cli] /Users/runner/work/flutter_rust_bridge/flutter_rust_bridge/target/x86_64-linux-android/debug/libflutter_rust_bridge_example.so -> ../android/app/src/main/jniLibs/x86_64/libflutter_rust_bridge_example.so

Reply:
* ios
[INFO  cargo_lipo::lipo] Creating universal library for flutter_rust_bridge_example